content/nldi.md

```diff
@@ -2,7 +2,6 @@
 author: David Blodgett
 date: 2017-04-22
 slug: nldi-intro
-draft: True
```

Comment:
In general though, I'd say that's the last thing to ever do, and should probably be done by the approver.